index.html

```diff
@@ -0,0 +1,93 @@
+        ><br />
+        <select
+          current
+          role="dropdown"
+          id="dropdown"
+          placeholder="select your current role"
+          style="width: 150px"
+        >
+          <option value="" disabled selected>Select an Option</option>
+          <option value="Student">Student</option>
+          <option value="Full Time Job">Full Time Job</option>
+          <option value="Full Time Learner">Full Time Learner</option>
+          <option value="Prefer not to say">Prefer not to say</option>
+          <option value="Other">Other</option>
+        </select>
+        <div>
+        <p>Would you love to be a Project Manager?</p>
+        <label for="Definitely">
+            <input type="radio" name="Manager" value="Definitely">Definitely</label></br>
+        <label for="Maybe">
+            <input type="radio" name="Manager" value="Maybe">Maybe</label></br>
+        <label for="Not sure"> 
+            <input type="radio" name="Manager" value="Not sure">Not sure</label>
+        </div></br>
+        <div>
+          <p>What would you like to see improve? (check all that applies)</p>
+          <label 
+       for="Improve Project"><br><input id="Front-end Projects" type="checkbox" value="Improve Project">Front-end Projects </label>
+       <label 
+       for="Improve Project"><br><input id="Back-end Projects" type="checkbox" value="Improve Project">Back-end Projects </label>
+       <label 
+       for="Improve Project"><br><input id="Data Visualization" type="checkbox" value="Improve Project">Data Visualization </label>
+       <label 
+       for="Improve Project"><br><input id="Challenges" type="checkbox" value="Improve Project">Challenges </label>
+       <label 
+       for="Improve Project"><br><input id="Open Source Communty" type="checkbox" value="Improve Project">Open Source Communty </label>
+       <label 
+       for="Improve Project"><br><input id="Gitter helps rooms" type="checkbox" value="Improve Project">Gitter helps rooms </label>
+       <label 
+       for="Improve Project"><br><input id="Videos" type="checkbox" value="Improve Project">Videos </label>
+       <label 
+       for="Improve Project"><br><input id="City Meetups" type="checkbox" value="Improve Project">City Meetups </label>
+       <label 
+       for="Improve Project"><br><input id="Wiki" type="checkbox" value="Improve Project">Wiki </label>
+       <label 
+       for="Improve Project"><br><input id="Forum" type="checkbox" value="Improve Project">Forum </label>
+       <label 
+       for="Improve Project"><br><input id="Additional Courses" type="checkbox" value="Improve Project">Additional Courses </label>
+      </div>
+
+      <p>Any comments or suggestions?</p>
+        <textarea rows = "5" cols = "50" placeholder="Enter your comments here.."></textarea></br>
+        <button type="submit" style="width: 390px; background-color:aqua;">submit</button> 
+    </form>
+</body>
+</main>
```

Comment:
I think its best practice if you wrap your body tag around the main tag instead of the other way round

Reply:
- [ ] After making a suggestion it is always nice to follow it up with a "why" explaining the importance of the changes that need to be made.
> First state the aimed problem ❗, politely offer a suggestion, and conclude with a reason or importance of making that change.

Reply:
[OPTIONAL]

> other way round, coz the body

Watch out for the typos, :eyes: we should try to use a formal language to stay professional :briefcase: